test/suite/test_gc01.py

```diff
@@ -69,7 +55,7 @@ def large_modifies(self, uri, value, ds, location, nbytes, nrows, commit_ts):
         session = self.session
         cursor = session.open_cursor(uri)
         session.begin_transaction()
-        for i in range(1, nrows):
+        for i in range(1, nrows + 1):
```

Comment:
This is an unusual usage so maybe a comment here about why we have to go to `nrows + 1`. Or alternatively, some other check could look for `nrows - 1` and this stays just `nrows`. I don't feel strongly though.

Reply:
I changed the range to `(0, nrows)` everywhere it was used.